hack/generate-asset-hashes.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd "${REPO_ROOT}"
+
+
+function generate_k8s_hashes() {
+  prefix=$1
+  max_patch=$2
+
+  cat > "${REPO_ROOT}/pkg/assets/assetdata/k8s-${prefix}.yaml" <<EOF
+# This file is generated by generate-asset-hashes.sh
+
+filestores:
+- base: https://dl.k8s.io/release/
+
+files:
+EOF
+
+  for ((patch = 0 ; patch < max_patch ; patch++ )); do
+    version="${prefix}.${patch}"
+    echo "k8s ${version}"
+
+    # We exclude some files that we don't currently need, to keep the size down
+    go run ./pkg/assets/assetdata/tools/cmd/generatefileassets \
+      --base https://dl.k8s.io/release/ \
+      --prefix "v${version}/" \
+      --exclude "**/s390x/**" \
```

Comment:
Should we also exclude `**/arm/**`

Reply:
Good call - it's easily confused with arm64.  Done!

Reply:
Very nice. Thanks!